CONTRIBUTING.md

```diff
@@ -90,13 +90,19 @@ Origin](#developers-certificate-of-origin-11) at the end of this file.*
   ├── general
   ├── linalg
   ├── mesh
-  └── miniapps
-      ├── common
-      ├── electromagnetics
-      ├── meshing
-      ├── nurbs
-      ├── performance
-      └── tools
+  ├── miniapps
+  │   ├── common
+  │   ├── electromagnetics
+  │   ├── meshing
+  │   ├── nurbs
+  │   ├── performance
+  │   └── tools
+  └── test
+      └── unittest
+          ├── data
+          ├── fem
+          ├── linalg
+          └── mesh
```

Comment:
I think we do not need to list the subdirectories below `unittest` here. We can just put `└── ...` under it to indicate that there are more subdirectories there, but there is no need to list them all here.